README.adoc

```diff
@@ -84,12 +84,41 @@ spec:
       password:
         secret: app_database_auth
         key: password
+      database:
+        secret: app_database_auth
+        key: dbname
+      port:
+        secret: app_database_auth
+        key: dbport
```

Comment:
I'm not sure if adding more values to this secret helps explain anything? If we do add anything then we should also make sure the comment above is updated? but remember it is just an example to show how the secrets manager works, it doesn't have to include everything.

Reply:
I tried to show that we can store the secret with a name that is different than the value it's listed in the yaml in the secret itself.